internal/session/repository_connection.go

```diff
@@ -81,6 +81,49 @@ func (r *ConnectionRepository) list(ctx context.Context, resources interface{},
 	return nil
 }
 
+func (r *ConnectionRepository) updateBytesUpBytesDown(ctx context.Context, conns ...*Connection) error {
+	const op = "session.(ConnectionRepository).updateBytesUpBytesDown"
+	if len(conns) == 0 {
+		return nil
+	}
+
+	updateMask := []string{"BytesUp", "BytesDown"}
+	_, err := r.writer.DoTx(
+		ctx,
+		db.StdRetryCnt,
+		db.ExpBackoff{},
+		func(read db.Reader, w db.Writer) error {
```

Comment:
```suggestion
		func(_ db.Reader, w db.Writer) error {
```
nit: the `read` parameter is not used so replacing it with the blank identifier makes that clearer.